qrcode_detection_package/main.py

```diff
@@ -1,35 +1,60 @@
 import sys
 import rclpy
 from rclpy.node import Node
+import cv2
+from std_msgs.msg import String
 
 from common_package_py.common_node import CommonNode
 
 
-class MinimalPublisher(CommonNode):
-    """A class that represents a minimal node.
-    """
-    
+class QRCodeScannerNode(CommonNode):
     def __init__(self, id: str):
-        """Creates a new MinimalPublisher node.
-
-        Args:
-            id (str): Unique node id
-        """
-        
         super().__init__(id)
-
+        self.qr_publisher = self.create_publisher(
+            String, 'qr_codes', 10)
+
+    def capture_image(self):
+        # For testing purposes the image is not taken from the camera but loaded a test file
+        # Path to the image file
+        image_path = '/home/ws/src/src/qrcode_detection_package/test_image/test.jpg'
```

Comment:
Hardcoden von dem Pfad sollte vermieden werden. Auch wenn es bei uns in Docker läuft.

Reply:
Ich hätte eine solche Funktionalität auch eher in einem test erwartet als das direkt in der Klasse zu hardcoden.
Für PoC ja, aber gepushed in dev weniger.